sherpa-onnx/csrc/online-recognizer-transducer-impl.h

```diff
@@ -83,8 +83,14 @@ class OnlineRecognizerTransducerImpl : public OnlineRecognizerImpl {
       : OnlineRecognizerImpl(config),
         config_(config),
         model_(OnlineTransducerModel::Create(config.model_config)),
-        sym_(config.model_config.tokens),
         endpoint_(config_.endpoint_config) {
+    if (!config.model_config.tokens_buf.empty()) {
+      sym_ = std::move(SymbolTable(config.model_config.tokens_buf, false));
```

Comment:
```suggestion
      sym_ = SymbolTable(config.model_config.tokens_buf, false);
```

No need to use `std::move()` here.